### ISSUE TITLE
Python Syntax Highlighting Seems Broken

### ISSUE THREAD

**Issue by sl8r000** [chriskempson/base16-vim#101]
# Summary
After running PluginUpdate today and getting on the latest version of base16 vim, python syntax highlighting seems to be off. Ruby still works fine:

<img width="1280" alt="screen shot 2016-07-04 at 4 26 26 pm" src="https://cloud.githubusercontent.com/assets/2441787/16570887/6a082044-4204-11e6-8e7c-f00a1e6ed968.png">

Specifically, for example, `self` is no longer highlighted. I also believe that the colorscheme for .py used to be more similar to that for .rb, so that (e.g.,) `class` should be purple and not red.

# Details

* I've already changed `colorscheme=base16-default` to `colorscheme=base16-default-dark` (as the former no longer exists, per a couple commits ago)
* This is in iTerm2, and I have the iTerm2 color scheme set (`base16-default.dark.256`). I also have `xterm-256color` set as well.
* I updated the base16-shell script too, and changed the path (since this also changed in a recent base16-shell commit).

**Comment by chriskempson** [comment]
Care to commit that python example to https://github.com/chriskempson/base16-code-examples? It makes it easier for me to checkout the file and start fixing the issue thanks 😄

**Comment by omnisci3nce** [comment]
Yep, exact same issue except I'm using base16-default.dark.itermcolors

**Comment by chriskempson** [comment]
@sl8r000 or just copy and paste your example here so I can submit it to https://github.com/chriskempson/base16-code-examples

**Comment by ethanrubio** [comment]
```python
import library

string = "base16"

fixnum = 0
_float = 0.00
array = list()
array = ['chris', 85]
_hash = {"test": "test"}

# this is a comment
class Person(object):
    def __init__(self, name):
        self.name = name

    def greet(self):
        return "hello"

person1 = Person(name="Chris")
print person1.greet(), "", person1.name, "\n"
```

I'm having the same issue as well. Here is the code example.

**Comment by jlesquembre** [comment]
@sl8r000 Which plugin are you using to highlighting python syntax? With the default vim syntax file ([link](https://github.com/neovim/neovim/blob/master/runtime/syntax/python.vim)), `self` (nor function arguments) are highlighted
